File: BEE 1154.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int sum = 0;
        int cnt = 0;
        while(true){
            int n = input.nextInt();
            if (n < 0){
                break;
            }
            sum += n;
            cnt += 1;
        }
        System.out.printf("%.2f\n", (((1.0) *sum) / cnt));
    }
}

File: BEE 1154.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var sum = 0;
var cnt = 0;

while(true){
    var n = parseInt(lines.shift());
    if (n < 0){
        break;
    }
    sum += n;
    cnt += 1;
}

console.log((sum / cnt).toFixed(2));
